doc/installing.rst

```diff
@@ -393,6 +393,22 @@ Some notes if you have problems installing the C code:
     - for some Mac users, you may also need to update your OS if you're using a
       particularly old version (e.g., 10.10 Yosemite)
 
+Some of the C code uses `OpenMP <https://www.openmp.org/>`_ to parallelize loops
+and take advantage of multiple cores/threads. This support is transparent and the code
+will work single-threaded if OpenMP is not available. GCC supports OpenMP
+out of the box, however the ``clang`` compiler that Apple's XCode provides does not. So
+for optimal performance on Apple hardware, you will want to install GCC via ``homebrew``
+or ``macports`` and specify its use when installing ``pypeit``. For example, if you installed
+GCC 12.x via ``homebrew``, you would get ``pypeit`` to use it by doing, for example:
+
+.. code-block:: console
+
+    CC=gcc-12 pip install "pypeit[pyqt5]"
+
+Basically, ``pypeit`` checks the ``CC`` environment variable for what compiler to use so configure
+that as needed to use your desired compiler. The ``pypeit_c_enabled`` script can be used to check if
+your compiler has OpenMP support.
```

Comment:
Nice.  Thanks for including this explanation, and I'm happy with this solution.  But, out of curiosity, do you know if there's a way to do this within the `setup.cfg` options?  I.e., could we enable something like `pip install "pypeit[pyqt5,openmp]"` and have that be the equivalent of the above?

Reply:
maybe? openmp isn't a python thing, though. it's compiler level. testing for it in a cross-platform way requires running a compiler so i don't see how to easily fold that into `setup.cfg`. it could go in `setup.py`, though. that's basically how the now-deprecated `astropy-helpers` package did it.

Reply:
it looks like `conda-forge` uses `clang` as the compiler for the `osx` platform. don't see an obvious way to specify `gcc`, but will look a little deeper...